docs/tutorials/tokens/erc1155.md

```diff
@@ -172,36 +172,14 @@ You can also approve other wallets to transfer tokens on your behalf with the `/
 - `operator`: The other account that is allowed to transfer tokens out of the wallet specified in the `key` field
 - `key`: The wallet address for the approval. If not set, it defaults to the address of the FireFly node submitting the transaction
 
-Here is an example request that would let the signing account `0x634ee8c7d0894d086c7af1fc8514736aed251528` transfer up to 10×10<sup>18</sup> (`10000000000000000000`) tokens from my wallet
+Here is an example request that would let the signing account `0x634ee8c7d0894d086c7af1fc8514736aed251528` transfer any amount of tokens from my wallet
 
 #### Request
 `POST` `http://127.0.0.1:5000/api/v1/namespaces/default/tokens/approvals`
 ```json
 {
   "approved": true,
```

Comment:
Could remove "approved" (here and some other places) since true is the default now.